### PR TITLE
Fixes #91: Move SSL options, basic auth & bearer token to constructor

### LINKED ISSUE #91: Move ssl_options & auth options into constructor

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,19 +49,22 @@ client = Kubeclient::Client.new uri
 It is also possible to use https and configure ssl with:
 
 ```ruby
-client = Kubeclient::Client.new 'https://localhost:8443/api/' , "v1beta3"
-client.ssl_options(
+ssl_options = {
```
Comment (simon3z) [inline README.md]: Can you make this consistent with `auth_options` below? (either both defined in `ssl_options` and `auth_options` or just passed to the constructor)

Comment (jimmidyson) [inline README.md]: Done.

**Comment by simon3z** [comment]
@jimmidyson have you tested this with `watch` as well? Other than that :+1:

**Comment by jimmidyson** [comment]
@simon3z of course i've tested it ;) just now. Works great.

**Comment by abonas** [comment]
merging based on @simon3z  review.

**Pull request merged**